js/script.js

```diff
@@ -0,0 +1,64 @@
+import { $taskForm, $taskTitle, $taskList, $btnCreate } from '../js/elements.js';
+
+let tasks = [];
+let taskId = 0;
+const $errorMessage = document.createElement("p");
+$errorMessage.className = "error-message";
+
+const openForm = () => {
+    $taskForm.classList.add("show");
+};
+
+const closeForm = () => {
+    $taskForm.classList.remove("show");
+};
+
+const addTask = () => {
+    const title = $taskTitle.value.trim();
+    if (title === "") {
+        showError("Please enter a task title.");
+        return;
+    }
+
+    const newTask = {
+        id: Date.now(), 
+        title: title,
+        createdAt: new Date().toLocaleDateString(),
+        completed: false,
+    };
+
+    tasks.push(newTask);
+    renderTasks();
+    $taskTitle.value = "";
+    closeForm();
+};
+
+const createTaskCard = task => {
+    const taskCard = document.createElement("div");
+    taskCard.className = "task-card";
+    taskCard.innerHTML = `
+        <p>${task.title}</p>
+        <p class="created-at">Created At: ${task.createdAt}</p>
+        <button class="btn-edit">Edit</button>
+        <button class="btn-delete">Delete</button>
+    `;
```

Comment:
For this project you can use template literals for rendering a card like this. But in a larger project this will create performance and security issue. If you want to know actual behavior you should use the DOM manipulation's general way.